scripts/codegen-operator

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+source $(dirname $0)/lib/debug_functions
+
+GOPATH="${GOPATH:-~/go}"
+GOFLAGS=""
+
+apt-get update -y || true
+apt-get install -y make mercurial cvs || dnf install make mercurial cvs
+mkdir -p $GOPATH/bin
+mkdir -p /root/go/bin
+
+export PATH=$PATH:$GOPATH/bin:/root/go/bin
+
+OP_FRAMEWORK="$GOPATH/src/github.com/operator-framework"
+
+mkdir -p $OP_FRAMEWORK
+pushd $OP_FRAMEWORK
+
+kind delete cluster || true
+kind create cluster
+
+git clone https://github.com/operator-framework/operator-sdk
```

Comment:
Why don't you use go get?
`go get -d github.com/operator-framework/operator-sdk`

Reply:
did that work for you?, it didn't for me, but let me try with your exact command.

This is used by make codegen-operator but I will update the README.md, you're right

Reply:
I tried that, but afterwards can't find operator-sdk installed anywhere (path or not...) 
```
root@cd8819185765:/go/src/github.com/submariner-io/submariner# find / -name "operator-sdk"
/go/pkg/mod/github.com/operator-framework/operator-sdk@v0.10.0/cmd/operator-sdk
/go/pkg/mod/cache/download/github.com/operator-framework/operator-sdk
/go/src/github.com/submariner-io/submariner/operators/go/submariner-operator/vendor/github.com/operator-framework/operator-sdk
root@cd8819185765:/go/src/github.com/submariner-io/submariner# /go/pkg/mod/github.com/operator-framework/operator-sdk@v0.10.0/cmd/operator-sdk
bash: /go/pkg/mod/github.com/operator-framework/operator-sdk@v0.10.0/cmd/operator-sdk: Is a directory
```

Reply:
outdated now